app/presenters/licences/view-licence-returns.presenter.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+/**
+ * Formats data for common licence data `/licences/{id}` page's
+ * @module ViewLicenceReturnsPresenter
+ */
+
+const { formatLongDate } = require('../base.presenter')
+
+/**
+ * Formats data for common licence data `/licences/{id}` page's
+ *
+ * @returns {Object} The data formatted for the view template
+ */
+function go (returnsData) {
+  const returns = _formatReturnToTableRow(returnsData.returns)
+
+  return {
+    activeTab: 'returns',
+    returnsUrl: 'return/internal',
```

Comment:
is this ever going to be anything else? even so why not just hard code it in the nunjucks file until we need to make it dynamic?

Reply:
I will hard code in the njk

Reply:
done